tools/api/generate_go_protobuf.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+from subprocess import check_output
+from subprocess import check_call
+
+import glob
+import os
+import shutil
+import sys
+import re
+
+targets = '@envoy_api//...'
+import_base = 'github.com/envoyproxy/go-control-plane'
+output_base = 'build_go'
+repo_base = 'go-control-plane'
+branch = 'master'
+mirror_msg = 'Mirrored from envoyproxy/envoy @ '
+user_name = 'Kuat Yessenov'
```

Comment:
Are we baking this in?

Reply:
I couldn't get past DCO blocking bot commits, so put my name if you don't mind.